administrative-sdk/connection/connection-controller.js

```diff
@@ -215,14 +215,15 @@ export default class Connection {
   handleResponse(response) {
     return response.text()
           .then(textResponse => {
-            if (!textResponse) {
+            if (response.headers.get('Content-type') === 'application/json; charset=utf-8') {
```

Comment:
I'm nut sure about the charset being there all the time and in the future. I'd just check for existence of substring application/json.